### PR TITLE
Trust Registry CLI commands

### PR DESCRIPTION
**Pull request by tmarkovski**
CLI commands for Trust Registry service

### CODE REVIEW

**Review by fundthmcalculus — APPROVED** [review]

File: cli/src/cli.yaml [hunk 1]
```diff
@@ -352,6 +352,125 @@ subcommands:
...
+        - search:
+            about: |
+              Search the registry for authoritative issuer and verifiers
+              using a custom query in a SQL format
+            args:
+              - query:
+                  long: query
+                  value_name: SQL query
+                  help: |
+                    Input SQL query to use to search the registry.
```
Comment (michaeldboyd) [inline cli/src/cli.yaml]: probably meant "input sql query to search the registry"

File: cli/src/cli.yaml [hunk 2]
```diff
@@ -352,6 +352,125 @@ subcommands:
...
+                    Default value is "SELECT * FROM c"
+                  takes_value: true
+                  required: true
+        - check-issuer:
+            about: |
+              Check the status of an issuer for a credential type within
+              a given governance framework.
+              Returns all historical data for the given input parameters
+            args:
+              - did-uri:
```
Comment (michaeldboyd) [inline cli/src/cli.yaml]: Why shouldn't we just call this `did` ? a `did-uri` seems like it's supposed to be a URI of the DID

**Review by michaeldboyd — APPROVED** [review]
Just a naming comment and a typo comment

File: cli/src/cli.yaml (hunk 1 above)
Comment (tmarkovski) [reply cli/src/cli.yaml]: Updated

File: cli/src/cli.yaml (hunk 2 above)
Comment (tmarkovski) [reply cli/src/cli.yaml]: Updated. Feel free to make PR with any changes related to naming, text, spelling, etc. Merging this.

**Pull request merged**